Example/Tests/Tests.swift

```diff
@@ -1,27 +1,148 @@
 import XCTest
 import QBRepository
+import RealmSwift
+
+class RealmrepositorysitoryTests: XCTestCase {
+
+    var repository: RealmRepository<QuickEmployee>!
 
-class Tests: XCTestCase {
-    
     override func setUp() {
         super.setUp()
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        let testRealm = try! Realm(configuration: Realm.Configuration(inMemoryIdentifier: "Test"))
+        repository = RealmRepository<QuickEmployee>(realm: testRealm)
     }
     
     override func tearDown() {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        repository.deleteAll { _ in }
+        let testRealm = try! Realm(configuration: Realm.Configuration(inMemoryIdentifier: "Test"))
+        repository = RealmRepository<QuickEmployee>(realm: testRealm)
+
         super.tearDown()
     }
     
-    func testExample() {
-        // This is an example of a functional test case.
+    func testGetAll() {
+        let testEmployees = createMockEmployees()
+        repository.create(testEmployees) { _ in }
+        repository.getAll{ (allObjects) in
+            XCTAssert(allObjects.count == testEmployees.count)
+        }
     }
-    
-    func testPerformanceExample() {
-        // This is an example of a performance test case.
-        self.measure() {
-            // Put the code you want to measure the time of here.
+
+
+    func testDeleteAll() {
+        let testEmployees = createMockEmployees()
```

Comment:
You do this in every test case, why not using the setUp method?

Reply:
i am doing this in two cases, for all other cases i want to insert specific data into the repo because the assertion is based on that